src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -605,6 +643,337 @@ def make_default_signatures(pretrained: t.Any) -> ModelSignaturesType:
     return {k: default_config for k in infer_fn}
 
 
+def import_model(
+    name: Tag | str,
+    model_name_or_path: str | os.PathLike[str],
+    *,
+    task_name: str | None = None,
+    task_definition: dict[str, t.Any] | TaskDefinition | None = None,
+    proxies: dict[str, str] | None = None,
+    revision: str = "main",
+    variant: str | None = None,
+    sync_with_hub_version: bool = False,
```

Comment:
I'm not sure if this should always be True. Maybe it can be True by default, but in some situation, user may want to have different tag version (e.g. user want more memorable version name)

Reply:
> I'm not sure if this should always be True. Maybe it can be True by default, but in some situation, user may want to have different tag version (e.g. user want more memorable version name)

The argument for more memorable version name should be set on the CLI. I don't think that we even have support for custom version on `save_model`? It is all generated by us right?

Reply:
I'm not sure. In diffusers we allow user to specify the model name, but use something like `diffusers_simple` to provide a way to auto generate the model name.

I think we should make `benotml.diffusers` and `bentoml.transformers` more generic and have more flexible apis for more general usage, and provide `benotml.diffusers_simple` and `bentoml.transformers_simple` to have a simplified api with some specified model in mind.